tests/lib/ocpp/v201/test_smart_charging_handler.cpp

```diff
@@ -321,7 +321,7 @@ TEST_F(ChargepointTestFixtureV201, K01FR09_IfTxProfileEvseHasNoActiveTransaction
     EXPECT_THAT(sut, testing::Eq(ProfileValidationResultEnum::TxProfileEvseHasNoActiveTransaction));
 }
 
-TEST_F(ChargepointTestFixtureV201, K01FR48FR19_NumberPhasesOtherThan1AndPhaseToUseSet_ThenProfileInvalid) {
```

Comment:
Hey @shankari we actually did not mean to remove these. The reason there are two FRs is because FR 19 and 20 is what implements FR48. This was discussed in the WG.